br/br-use-overview.md

```diff
@@ -0,0 +1,101 @@
+
+### How to manage backup data?
+
+br command-line tool provides only basic backup and restore features, and does not support backup management. Therefore, you need to decide how to manage backup data on your own, which might involve the following questions?
+
+* Which backup storage system should I choose?
+* In which directory should I place the backup data during a backup task?
+* In what way should I organize the directory of the full backup data and log backup data?
+* How to handle the historical backup data in the storage system?
+
+The following sections will answer these questions one by one.
+
+**Choose a backup storage system**
+
+It is recommended that you store backup data to Amazon S3, Google Cloud Storage (GCS), or Azure Blob Storage. Using these systems, you do not need to worry about the backup capacity and bandwidth allocation.
+
+If the TiDB cluster is deployed in a self-built data center, the following practices are recommended:
+
+* Build [MinIO](https://docs.min.io/docs/minio-quickstart-guide.html) as the backup storage system, and use the S3 protocol to back up data to MinIO.
+* Mount Network File System (NFS, such as NAS) disks to br command-line tool and all TiKV instances, and use the POSIX file system interface to write backup data to the corresponding NFS directory.
+
+> **Note:**
+>
+> If no NFS is mounted to the br command-line tool or TiKV node, or if you use a storage system that supports Amazon S3, GCS, or Azure Blob Storage protocols, the data backed up by br command-line tool is generated at each TiKV node. **Note that this is not the recommended way to deploy the br command-line tool**, because the backup data are scattered in the local file system of each node. Collecting the backup data might result in data redundancy and operation and maintenance problems. Meanwhile, if you restore data directly before collecting the backup data, you will encounter the `SST file not found` error.
+
+**Organize the backup data directory**
+
+* Store the snapshot backup and log backup in the same directory for unified management, for example, `backup-${cluster-id}`.
+* Store each snapshot backup in a directory with the backup date included, for example, `backup-${cluster-id}/fullbackup-202209081330`.
+* Store the log backup in a fixed directory, for example, `backup-${cluster-id}/logbackup`. The log backup program creates a subdirectory in the `logbackup` directory every day to distinguish the data backed up each day.
+
+**Handle historical backup data**
+
+Assume that you have specified a **backup retention period**. Backup data will be retained only for this period, for example, 7 days. The concept **backup retention period** will also be mentioned in backup tutorials.
```

Comment:
```suggestion
Assume that you need to set the life cycle for each backup data, for example, 7 days. The concept **backup retention period** will also be mentioned in backup tutorials.
```

Reply:
This change should be synced to Chinese as well.